src/compute-pr-actions.ts

```diff
@@ -394,6 +379,35 @@ export function process(prInfo: PrInfo | BotEnsureRemovedFromProject | BotNoPack
     return context;
 }
 
+function makeStaleness(now: string, author: string, otherOwners: string[]) { // curried for convenience
+    return (kind: StalenessKind, since: Date,
+            freshDays: number, attnDays: number, nearDays: number,
+            doneColumn: ColumnName | "CLOSE") => {
+        const days = daysSince(since, now);
+        const state = days <= freshDays ? "fresh" : days <= attnDays ? "attention" : days <= nearDays ? "nearly" : "done";
+        const kindAndState = `${kind}:${state}`;
+        const explanation = Comments.StalenessExplanations[kindAndState];
+                const comment = Comments.StalenessComment(author, otherOwners)[kindAndState];
+        const doTimelineActions = (context: Actions) => {
+            if (comment !== undefined) {
+                const tag = state === "done" ? kindAndState
+                    : `${kindAndState}:${since.toISOString().replace(/T.*$/, "")}`;

```

Comment:
Am I right that the date in the tag isn't consumed yet?

Reply:
The tag *contains* the date.  The reasoning is that when a staleness timeline starts, the tag for the comment would be based on the day it started, which means that if a PR gets out of the staleness timeline and later gets back into it, there would be a date for that, and therefore a second post.

Reply:
Sounds good, where does that happen?

Reply:
Either I'm not following you or the bogus replacement line was confusing...?  (It's used in the comment that is pushed right after this bit.)

Reply:
I'm searching for the code that pulls the date out of an existing comment and uses it for the described behavior.

Reply:
The date string is in the `tag`, `getMutationsForComments` (in `execute-pr-actions`) uses `comment.make` to embed the tag as an html comment in the posted comment, and it also searches for an existing comment with the same tag to decide whether it should be a new comment, or editing an existing one, or nothing at all.

Reply:
Ohhh sorry it's an identifier to discriminate comments originating from different countdowns (re-reading your explanation and considering the purpose of `tag` it's obvious). Thanks!

Reply:
Yes, exactly.